### DISCUSSION TITLE
Exporting only certain logs to Prometheus

### DISCUSSION THREAD

**Discussion by andrii-bodnar** [martin-helmich/prometheus-nginxlog-exporter#271] [Q&A] [answered]
Hey everyone,

Is there any way to export logs corresponding to some list of limited request URIs instead of exporting metrics for all nginx logs?

In my case, I need metrics only for a few amount of requests and do not need all the requests data in my Prometheus instance.

**Comment by stempler** [comment, marked as answer]
You can adapt the scrape configuration in Prometheus to only keep time series with certain labels / label values.
See [metric_relabel_configs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#metric_relabel_configs) in the Prometheus docs. [Here](https://medium.com/quiq-blog/prometheus-relabeling-tricks-6ae62c56cbda) is an article with some examples.

**Reply by andrii-bodnar** [reply]
@stempler thanks a lot for quick response!

I will check provided links. It looks like it should work in my case.
